public/js/app.js

```diff
@@ -19,4 +21,25 @@ weatherForm.addEventListener('submit', (event) => {
             }
         })
     })
+})
+
+currLocWeather.addEventListener('click', (event) => {
+    event.preventDefault()
+    if (!navigator.geolocation) {
+        return alert('Geolocation is not supported by your browser.')
+    }
+    navigator.geolocation.getCurrentPosition((position) => {
```

Comment:
wrap to promise, use `async-await`

Reply:
https://gist.github.com/alexandr-senyuk/ccca4252f1064fbb4f7101f347fbfc21

Reply:
Fixed.

Also moved the code where the 'html' for forecast is created into a separate method, since this code is duplicated.